<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialogDeveloper</class>
 <widget class="QWidget" name="PreferencesDialogDeveloper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>739</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="developer">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Developer Mode</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>User interface</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Where to load user interface XML (ui files), translations and resources from. Productive systems fetch them from the server.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Load mode</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QRadioButton" name="uiLoadModeLocalFile">
              <property name="text">
               <string>Local file</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="uiLoadModeNetwork">
              <property name="text">
               <string>Network</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Translations directory:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="FileChooser" name="uiFormTranslationsDir">
             <property name="selectionMode">
              <enum>FileChooser::SelectExistingDir</enum>
             </property>
             <property name="title">
              <string>Select a directory holding UI form translations</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="1">
            <widget class="FileChooser" name="uiFormsDir">
             <property name="selectionMode">
              <enum>FileChooser::SelectExistingDir</enum>
             </property>
             <property name="title">
              <string>Select a directory holding UI forms</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Form directory:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Resources directory:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="FileChooser" name="uiFormResourcesDir">
             <property name="selectionMode">
              <enum>FileChooser::SelectExistingDir</enum>
             </property>
             <property name="title">
              <string>Select a directory holding UI form resources</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>&amp;Data Processing</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Load mode</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QRadioButton" name="dataLoadModeLocalFile">
              <property name="text">
               <string>Local file</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="dataLoadModeNetwork">
              <property name="text">
               <string>Network</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Data directory:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="FileChooser" name="dataLoaderDir">
             <property name="selectionMode">
              <enum>FileChooser::SelectExistingDir</enum>
             </property>
             <property name="title">
              <string>Select a directory which contains local XML data</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Usually requests and answers containing data forms are sent between client and server. For testing the UI some mock data can be retrieved from the file system and saved to the file system .</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="debug">
        <property name="text">
         <string>Enable verbose &amp;debug output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoLogin">
        <property name="text">
         <string>Login &amp;automatically when starting the client</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileChooser</class>
   <extends>QWidget</extends>
   <header>FileChooser.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>developer</tabstop>
  <tabstop>uiLoadModeLocalFile</tabstop>
  <tabstop>uiLoadModeNetwork</tabstop>
  <tabstop>uiFormsDir</tabstop>
  <tabstop>uiFormTranslationsDir</tabstop>
  <tabstop>uiFormResourcesDir</tabstop>
  <tabstop>dataLoadModeLocalFile</tabstop>
  <tabstop>dataLoadModeNetwork</tabstop>
  <tabstop>dataLoaderDir</tabstop>
  <tabstop>debug</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
